packages/design-system-react/jest.config.js

```diff
@@ -14,6 +14,9 @@ module.exports = merge(baseConfig, {
   // The display name when running multiple projects
   displayName,
 
+  // Add coverage ignore patterns
+  coveragePathIgnorePatterns: ['index.ts'],
```

Comment:
Ignoring index files from test coverage as they only re-export components. We could add tests for these in the future, but will ignore them for now.